src/Tizen.NUI/src/public/BaseComponents/ViewEvent.cs

```diff
@@ -830,6 +830,8 @@ private void OnRelayout(IntPtr data)
             {
                 onRelayoutEventHandler(this, null);
             }
+
+            Layout?.RequestLayout();
```

Comment:
NUI layout is works before this OnRelayout callback is called.
This code works for dimension dependency?

Reply:
This code is added to calculate NUI Layout again when ImageView's size is changed from Dali.
However, this code is too simple so it may cause redundant Layout calculation.
I will update the code.